src/plugins/visualizations/public/wizard/type_selection/type_selection.tsx

```diff
@@ -95,22 +99,48 @@ class TypeSelection extends React.Component<TypeSelectionProps, TypeSelectionSta
     return (
       <React.Fragment>
         <EuiModalHeader>
-          <EuiModalHeaderTitle>
-            <FormattedMessage
-              id="visualizations.newVisWizard.title"
-              defaultMessage="New Visualization"
-            />
-          </EuiModalHeaderTitle>
+          <EuiFlexGroup direction="column" gutterSize="m">
+            <EuiFlexItem>
+              <EuiModalHeaderTitle>
+                <FormattedMessage
+                  id="visualizations.newVisWizard.title"
+                  defaultMessage="New Visualization"
+                />
+              </EuiModalHeaderTitle>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiCallOut
+                title="Introducing the new Visualization Builder"
```

Comment:
shouldn't the title be localized?

Reply:
@virajsanghvi  updated the title to include localization